src/Features/Core/Portable/CodeFixesAndRefactorings/FeaturesFixAllGetFixesService.cs

```diff
@@ -5,20 +5,17 @@
 using System;
 using System.Composition;
 using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
-using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions;
 
-[ExportWorkspaceServiceFactory(typeof(IFixAllGetFixesService), ServiceLayer.Host), Shared]
-internal sealed class FeaturesFixAllGetFixesService : AbstractFixAllGetFixesService, IWorkspaceServiceFactory
+[ExportWorkspaceService(typeof(IFixAllGetFixesService), ServiceLayer.Default), Shared]
```

Comment:
exporting at the 'Host' layer meant taht thsi "default" impl ended up overriding the "Editor" impl, so we got no previews and (because of hte bug about returning the old solution), nothing changed.